cypress/fixtures/flows/dashboard-text.json

```diff
@@ -295,7 +295,7 @@
                 "t": "set",
                 "p": "ui_update.fontSize",
                 "pt": "msg",
-                "to": "28px",
+                "to": "28",
```

Comment:
Change this from 28px to 28 as we don't define the unit when setting the font size via the node-red widget editor under "Text Size"

<img width="380" alt="Screenshot 2024-08-12 at 13 45 07" src="https://github.com/user-attachments/assets/fca99317-7ae4-4e8d-996a-76e4b67252b4">